configs/ote/template_paths.yml

```diff
@@ -0,0 +1,26 @@
+'mobilenet_v2-2s_ssd-256x256': ./custom-object-detection/mobilenet_v2-2s_ssd-256x256/template.yaml
+'mobilenet_v2-2s_ssd-384x384': ./custom-object-detection/mobilenet_v2-2s_ssd-384x384/template.yaml
+'mobilenet_v2-2s_ssd-512x512': ./custom-object-detection/mobilenet_v2-2s_ssd-512x512/template.yaml
+'mobilenetV2_ATSS': ./custom-object-detection/mobilenetV2_ATSS/template.yaml
+'mobilenetV2_SSD': ./custom-object-detection/mobilenetV2_SSD/template.yaml
+'resnet50_VFNet': ./custom-object-detection/resnet50_VFNet/template.yaml
+'face-detection-0200': ./face-detection/face-detection-0200/template.yaml
+'face-detection-0202': ./face-detection/face-detection-0202/template.yaml
+'face-detection-0204': ./face-detection/face-detection-0204/template.yaml
+'face-detection-0205': ./face-detection/face-detection-0205/template.yaml
+'face-detection-0206': ./face-detection/face-detection-0206/template.yaml
+'face-detection-0207': ./face-detection/face-detection-0207/template.yaml
+'horizontal-text-detection-0001': ./horizontal-text-detection/horizontal-text-detection-0001/template.yaml
+'person-detection-0200': ./person-detection/person-detection-0200/template.yaml
+'person-detection-0201': ./person-detection/person-detection-0201/template.yaml
+'person-detection-0202': ./person-detection/person-detection-0202/template.yaml
+'person-detection-0203': ./person-detection/person-detection-0203/template.yaml
+'person-vehicle-bike-detection-2000': ./person-vehicle-bike-detection/person-vehicle-bike-detection-2000/template.yaml
+'person-vehicle-bike-detection-2001': ./person-vehicle-bike-detection/person-vehicle-bike-detection-2001/template.yaml
+'person-vehicle-bike-detection-2002': ./person-vehicle-bike-detection/person-vehicle-bike-detection-2002/template.yaml
+'person-vehicle-bike-detection-2003': ./person-vehicle-bike-detection/person-vehicle-bike-detection-2003/template.yaml
+'person-vehicle-bike-detection-2004': ./person-vehicle-bike-detection/person-vehicle-bike-detection-2004/template.yaml
+'vehicle-detection-0200': ./vehicle-detection/vehicle-detection-0200/template.yaml
+'vehicle-detection-0201': ./vehicle-detection/vehicle-detection-0201/template.yaml
+'vehicle-detection-0202': ./vehicle-detection/vehicle-detection-0202/template.yaml
+'vehicle-detection-0203': ./vehicle-detection/vehicle-detection-0203/template.yaml
```

Comment:
What's the reason to have that kind of list?

Reply:
This file is required for `test_ote_training.py` file. It is pointed as a pytest parameter `--template-paths` (see the file `tests/conftest.py`). The parameter allows to point for a model name the realtive path where template file is placed.

Note that during tests development we used other `template_paths.yml` files as the parameter.

Reply:
This file is removed from the current PR.

Reply:
Such kind of lists are error-prone and hard to support. At the same time it does not match the way it's done in OTE, which might cause another point of confusion.

Reply:
Ok! At the moment I remove this file from this PR.
I think I will fix this your comment when I will make PR with my `test_ote_training.py`.